Examples/Tests/electrostatic_sphere/inputs_rz_uniform_weighting

```diff
@@ -0,0 +1,45 @@
+max_step = 30
+amr.n_cell = 32 64
+amr.max_level = 0
+amr.blocking_factor = 8
+amr.max_grid_size = 128
+geometry.dims     = RZ
+geometry.prob_lo  =  0.  -0.5
+geometry.prob_hi  =  0.5  0.5
+boundary.field_lo = none pec
+boundary.field_hi = pec pec
+warpx.const_dt = 1e-6
+warpx.do_electrostatic = labframe
+warpx.use_filter = 0
+
+particles.species_names = electron
+
+algo.field_gathering = momentum-conserving
+
+# Order of particle shape factors
+algo.particle_shape = 1
+
+my_constants.n0 = 1.49e6
+my_constants.R0 = 0.1
+
+electron.charge = -q_e
+electron.mass = m_e
+electron.injection_style = "NUniformPerCell"
+electron.num_particles_per_cell_each_dim = 6 2 2
```

Comment:
Note here that more particles are needed in this test compared to the non-uniformly weighted case to get to the same tolerance. This is because the error is being checked on axis which has fewer particles when doing uniform weighting.